src/components/GPSButton/GPSButton.tsx

```diff
@@ -79,14 +81,17 @@ const GPSButton: FunctionComponent<GPSButtonProps> = ({
     <StyledButton
       className={className}
       data-testid="gpsButton"
-      icon={loading ? <LoadingIndicator color="black" /> : <GPSIcon />}
+      iconLeft={loading ? <LoadingIndicator color="black" /> : <GPSIcon />}
       aria-label="Huidige locatie"
       iconSize={20}
       onClick={onClick}
-      size={44}
+      size={144}
       variant="blank"
       type="button"
-    />
+    >
+      {' '}
+      Mijn locatie
```

Comment:
Is dit iets wat misschien in de toekomst uit een translation file moet komen? Zodat meerdere clients een andere kunnen kiezen met 'Mijn locatie' als default.